timesketch/frontend-ng/src/components/Visualization/VisualizationEditor.vue

```diff
@@ -347,8 +373,10 @@ export default {
       this.selectedYTitle = this.yTitle
     },
     saveVisualization() {
+      if (this.selectedChartTitle === "New Visualization") {
+        this.selectedChartTitle = prompt("Please enter a name for the visualization:")
```

Comment:
if cancel is clicked during this prompt it results in a null value being assigned to selectedChartTitle which causes an error when rendering the LHS menu.  I'll add a fix/check for this.